apps/bfd-server/bfd-server-launcher/src/main/java/gov/cms/bfd/server/launcher/DataServerLauncherApp.java

```diff
@@ -81,6 +82,10 @@ public final class DataServerLauncherApp {
   public static final String HTTP_ACCESS_RESPONSE_OUTPUT_SIZE_IN_BYTES =
       "http_access_response_output_size_in_bytes";
 
+  /** MDC key for the response duration per kb. */
+  public static final String MDC_HTTP_ACCESS_RESPONSE_DURATION_PER_KB =
+      "mdc_http_access_response_duration_per_kb";
```

Comment:
'MDC' and 'mdc' should not be part of the constant name or the actual string.

Reply:
fixed